jvm-libs/linea/clients/eth-logs-searcher/src/test/kotlin/linea/ethapi/LogsSearcherTest.kt

```diff
@@ -81,6 +82,57 @@ class LogsSearcherTest {
     assertThat(result.endBlockNumber).isEqualTo(450UL)
   }
 
+  @Test
+  fun `should return logs within block range in face of temporary errors`() {
```

Comment:
This test would fail by timeout if the newly added [condition](https://github.com/Consensys/linea-monorepo/pull/1330/files#diff-b17bfc2505707e4971d05a6bf69d40bf089c733efbcd5fa4f8194056744a05bfR129:~:text=if%20(chunk%20%3D%3D%20null%20%7C%7C%20lastSearchedChunk.get()%20%3D%3D%20chunk) is absent